assets/src/hooks/useDetours.ts

```diff
@@ -170,6 +179,13 @@ export const useDraftDetours = (socket: Socket | undefined) => {
     })
   }
 
+  const handleDeleted = (detour_id: number) => {
```

Comment:
nit: as much as I love snake_case, we should be using camelCase for variables on the TS side.

I'm honestly baffled we don't have a lint for this.